shenyu-admin/src/main/java/org/apache/shenyu/admin/service/converter/AbstractSelectorHandleConverter.java

```diff
@@ -68,7 +68,7 @@ public String handler(final String handle, final List<CommonUpstream> aliveList)
      */
     @Override
     public <T extends CommonUpstream> List<T> updateStatusAndFilter(final List<T> existList, final List<? extends CommonUpstream> aliveList) {
-        if (CollectionUtils.isEmpty(aliveList) || CollectionUtils.isEmpty(existList)) {
+        if (aliveList == null) {
```

Comment:
use CollectionUtils.isEmpty(?

Reply:
If the aliveList is an empty array, follow-up logic is required

Reply:
Objects.isNull

Reply:
i will modify him